example/profile.php

```diff
@@ -25,22 +35,46 @@
 <html>
    <head>
         <meta charset="utf-8">
-        <title>PROFILE PAGE</title>
+        <title>YOTI PROFILE</title>
+
+        <link rel="stylesheet" type="text/css" href="css/style.css">
    </head>
    <body>
         <?php if (!empty($errorMsg)) : ?>
             <p><strong><?php echo $errorMsg ?></strong></p>
         <?php else: ?>
-            <strong>Given Name(s)</strong> <?php echo $profile->getGivenNames() ?><br>
-            <strong>Family Name</strong> <?php echo $profile->getFamilyName() ?><br>
-            <strong>Phone</strong> <?php echo $profile->getPhoneNumber() ?><br>
-            <strong>Email</strong> <?php echo $profile->getEmailAddress() ?><br>
-            <strong>Date Of Birth</strong> <?php echo $profile->getDateOfBirth() ?><br>
-            <strong>Address</strong> <?php echo $profile->getPostalAddress() ?><br>
-            <strong>Gender</strong> <?php echo $profile->getGender() ?><br>
-            <strong>Nationality</strong> <?php echo $profile->getNationality() ?><br>
-            <strong>Photo</strong><br> <img src="data:image/x-icon;base64,<?php echo $selfie ?>" />
-            <br>
+            <h1>Yoti User Profile</h1>
+            <dl>
+                <dt>Given Name(s)</dt>
+                <dd><?php echo $profile->getGivenNames() ?></dd>
+
+                <dt>Family Name</dt>
+                <dd><?php echo $profile->getFamilyName() ?></dd>
+
+                <dt>Phone</dt>
+                <dd><?php echo $profile->getPhoneNumber() ?></dd>
+
+                <dt>Email</dt>
+                <dd><?php echo $profile->getEmailAddress() ?></dd>
+
+                <dt>Date Of Birth</dt>
+                <dd><?php echo $profile->getDateOfBirth() ?></dd>
+
+                <dt>Address</dt>
+                <dd><?php echo $profile->getPostalAddress() ?></dd>
+
+                <dt>Gender</dt>
+                <dd><?php echo $profile->getGender() ?></dd>
+
+                <dt>Nationality</dt>
+                <dd><?php echo $profile->getNationality() ?></dd>
+
+                <dt>Selfie as base64 data</dt>
+                <dd><img src="<?php echo $base64Selfie ?>" /></dd>
```

Comment:
I feel like the `base64Selfie` should be inside the `$profile` object, but I can also understand why you want to have it in a separate helper.